apps/wallet-mobile/src/features/Send/useCases/StartMultiTokenTx/StartMultiTokenTxScreen.tsx

```diff
@@ -24,7 +23,7 @@ export const StartMultiTokenTxScreen = () => {
 
   const {targets, selectedTargetIndex, receiverChanged, memo, memoChanged, addressChanged} = useSend()
   const {address, amounts} = targets[selectedTargetIndex].entry
-  const shouldOpenSelectToken = Object.keys(amounts).length === 0
+  const shouldOpenAddToken = Object.keys(amounts).length === 0
```

Comment:
my bad here, it should be using `Amounts.toArray` helper, amounts must be manipulated with this helper always.